### PR TITLE
justin-h-im/about page

### LINKED ISSUE #13: About us
image

about us page

ignore the image at the bottom that overlaps

ignore the text witl in the center of the O as well

remember to use p tags, if you want to change the font size, look it up on tailwind!!

### PR DESCRIPTION
**Pull request by justin-h-im**
Created basis for the About Us title page. 
Images for Code Changes: 
image
image

Images for Product on Local Host:
image

### CODE REVIEW

File: src/app/about/page.js [hunk 1]
```diff
@@ -1,4 +1,17 @@
-const Page = () => {
-  return <div>Hello World</div>;
+import "@/app/layout.js";
```
Comment (stanleylew5) [inline src/app/about/page.js:1]:
```suggestion

```

Comment (stanleylew5) [inline src/app/about/page.js]:
```suggestion

```

Comment (stanleylew5) [reply src/app/about/page.js]: not needed as layout.js is already applied

File: src/app/about/page.js [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+        </div>
+        <div className="mb-1 text-9xl text-center text-purple-200">
+          <p>US</p>
+        </div>
+      </div>
+    </div>
+  );
 };
-export default Page;
+export default About;
```
Comment (stanleylew5) [inline src/app/about/page.js]: change the exporting back to page and move your code to an About.jsx file

File: tailwind.config.js [hunk 3]
```diff
@@ -18,6 +21,9 @@ module.exports = {
         },
       },
     },
+    backgroundColor: (theme) => ({
+      purplecustom: "#2E0144",
+    }),
```
Comment (stanleylew5) [inline tailwind.config.js]:
```suggestion
   
```

Comment (stanleylew5) [reply tailwind.config.js]: not needed, if you wanted to use this color as a background color use one of the already defined colors with tailwind

the class would be bg-witl-purple-300 (check above for already defined colors)

File: src/app/about/page.js [hunk 4]
```diff
@@ -1,4 +1,17 @@
-const Page = () => {
-  return <div>Hello World</div>;
+import "@/app/layout.js";
+
+const About = () => {
+  return (
+    <div className="bg-purplecustom min-h-screen flex items-center justify-center">
```
Comment (stanleylew5) [inline src/app/about/page.js]:
```suggestion
    <div className="bg-witl-purple-300 min-h-screen flex items-center justify-center">
```

File: src/app/about/page.js [hunk 5]
```diff
@@ -1,4 +1,17 @@
...
+      <div className="flex flex-col tracking-widest">
+        <div className="mb-1 text-9xl text-center text-purple-200">
```
Comment (stanleylew5) [inline src/app/about/page.js]: use witl colors

File: src/app/about/page.js [hunk 6]
```diff
@@ -1,4 +1,17 @@
...
+          <p>ABOUT</p>
+        </div>
+        <div className="mb-1 text-9xl text-center text-purple-200">
```
Comment (stanleylew5) [inline src/app/about/page.js]: use witl colors

**Review by stanleylew5 — CHANGES_REQUESTED** [review]

**Comment by justin-h-im** [comment]
Updated work to put everything into a jsx file, corrected colors, merged with dev on this branch.

Code Screenshots:
image
image

Output Screenshots:
image

**Pull request merged**